tap_twitter_ads/schemas/campaigns.json

```diff
@@ -8,10 +8,6 @@
     "name": {
       "type": ["null", "string"]
     },
-    "start_time": {
```

Comment:
Why we remove this?

Reply:
Added back `start_time` and `end_time` in the schema of campaigns.

Reply:
Actually, `start_time` and `end_time` fields are deprecated. ([link](https://twittercommunity.com/t/ads-api-version-11/168814#changed-moving-start-end-times-to-line-item-level-6)). But, I have removed these fields from [#25](https://github.com/singer-io/tap-twitter-ads/pull/25) instead of this PR.